addons/web_editor/static/src/js/backend/field_html.js

```diff
@@ -53,7 +53,33 @@ var FieldHtml = basic_fields.DebouncedField.extend(TranslatableFieldMixin, Dynam
         wysiwyg_change: '_onChange',
         wysiwyg_attachment: '_onAttachmentChange',
     },
+    /**
+     * @override
+     * @param {ModelFieldSelectorPopover} modelSelector
+     * @returns {Promise<ModelFieldSelectorPopover>}
+     */
+    insertAndPositionModelSelector: async function (modelSelector) {
+        // Let the default positioning do its thing.
```

Comment:
That's not enterirely correct IMO. You don't care about the positioning, do you ? You're going to reposition it anyway. What you care about is that it has been appended to the DOM. But that's only because the appending is currently done in the super call while I think it should be done around the call site, cf previous comment.